src/core_plugins/kibana/index.js

```diff
@@ -52,9 +53,16 @@ module.exports = function (kibana) {
           const configuredUrl = server.config().get('tilemap.url');
           const isOverridden = typeof configuredUrl === 'string' && configuredUrl !== '';
           const tilemapConfig = serverConfig.get('tilemap');
```

Comment:
why are vectorMappConfig and mapConfig cloning the server response but the tilemapConfig is not ?